spec/classes/init_spec.rb

```diff
@@ -256,6 +256,16 @@
           :vas_conf_vasd_workstation_mode_users_preload         => 'usergroup',
           :vas_conf_vasd_workstation_mode_group_do_member       => 'true',
           :vas_conf_vasd_workstation_mode_groups_skip_update    => 'true',
+          :vas_conf_vasd_username_attr_name			=> 'userprincipalname',
+	  :vas_conf_vasd_groupname_attr_name                    => 'groupprincipalname',
+	  :vas_conf_vasd_uid_number_attr_name                   => 'employeID',
+	  :vas_conf_vasd_gid_number_attr_name                   => 'primaryGroupID',
+	  :vas_conf_vasd_gecos_attr_name                        => 'displayName',
+	  :vas_conf_vasd_home_dir_attr_name                     => 'homeDirectory',
+	  :vas_conf_vasd_login_shell_attr_name                  => 'loginShell',
+	  :vas_conf_vasd_group_member_attr_name                 => 'groupMembershipSAM',
+	  :vas_conf_vasd_memberof_attr_name                     => 'memberOf',
+	  :vas_conf_vasd_unix_password_attr_name                => 'userPassword',
```

Comment:
None of my editors see that problem - tested with nano, vi and kate. 
Any ideas?